plutus-scb/src/Plutus/SCB/Webserver/Handler.hs

```diff
@@ -87,7 +91,13 @@ getChainReport = do
     relatedMetadata <-
         mconcat <$>
         traverse
```

Comment:
can be a bit cleaner with `for`:
```
for wallets $ \wallet -> do ...
```

Reply:
Need a hlint rule for `traverse <parenthesized multiline function> <variable>`

Reply:
Yup, nicer. 👍